website/orders/api/v1/views.py

```diff
@@ -302,6 +315,37 @@ def patch(self, request, **kwargs):
         return Response(status=status.HTTP_200_OK, data=ShiftSerializer(shift, context={"request": request}).data)
 
 
+class ShiftFinalizeAPIView(APIView):
+    """Shift Finalize API View."""
+
+    schema = CustomAutoSchema(response_schema={"$ref": "#/components/schemas/Shift"})
+    permission_required = "orders.can_manage_shift_in_venue"
+    permission_classes = [HasPermissionOnObject]
```

Comment:
And IsOnBakerList right #147

Reply:
Yes, let's fix this after merging #147 or let's fix it in #147 after merging this pull request.

Reply:
#147 is now approved so do your thing

Reply:
Already on it!